File: Lista de Usuários Windows/USERSWINDOWNS.java
import java.io.BufferedReader;
import java.io.InputStreamReader;

public class ListWindowsUsers {

    public static void main(String[] args) {
        try {
            // Comando para listar os usuários no Windows
            String command = "net user";

            // Executa o comando
            Process process = Runtime.getRuntime().exec(command);

            // Lê a saída do comando
            BufferedReader reader = new BufferedReader(new InputStreamReader(process.getInputStream()));
            String line;

            System.out.println("Contas de usuário no Windows:");
            while ((line = reader.readLine()) != null) {
                // Filtra as linhas que contêm os nomes dos usuários
                if (!line.trim().isEmpty() && !line.startsWith("-----") && !line.startsWith("Os comandos")) {
                    System.out.println(line.trim());
                }
            }

            // Fecha o leitor
            reader.close();

        } catch (Exception e) {
            e.printStackTrace();
        }
    }
}

File: Lista de Usuários Windows/USERSWINDOWNS.py
import subprocess

def users_win():
    try:

        comando = ["net", "user"]
        resultado = subprocess.run(comando, capture_output=True, text=True, shell=True, check=True)

        saida = resultado.stdout.splitlines()

        usuarios = [
            line.strip() for line in saida
            if line.strip() and not line.startswith("---") and not line.lower().startswith("the command")
        ]

        if usuarios:
            print("Contas de usuários no Windows:")
            for usuario in usuarios:
                print(usuario)
        else:
            print("Nenhuma conta de usuário encontrada.")

    except subprocess.CalledProcessError as e:
        print(f"Erro ao executar o comando: {e}")
    except Exception as e:
        print(f"Ocorreu um erro: {e}")

users_win()
